bitnami/Makefile

```diff
@@ -0,0 +1,30 @@
+NAME=timescaledb
+ORG=timescale
+PG_VER=pg10
+VERSION=$(shell awk '/^ENV TIMESCALEDB_VERSION/ {print $$3}' Dockerfile)
+
+default: image
+
+.build_$(VERSION)_$(PG_VER): Dockerfile
+ifeq ($(PG_VER),pg9.6)
+	docker build --build-arg PG_VERSION=9.6.11 -t $(ORG)/$(NAME):latest-$(PG_VER)-bn .
+	docker tag $(ORG)/$(NAME):latest-$(PG_VER)-bn $(ORG)/$(NAME):latest-bn
```

Comment:
I think the suffix should be `bitnami` instead of `bn` for clarity.

Reply:
Thx. Done